internal/app/parse.go

```diff
@@ -130,3 +130,15 @@ func (a *App) excludeFilesFromInput(inputFiles []string, excludeFiles []string)
 	}
 	return filtered
 }
+
+func (a *App) aggregateExcludeFiles(excludeFiles []string) ([]string, error) {
+	var aggregatedExcludeFiles []string
+	for _, excludeGlob := range excludeFiles {
+		excludeFiles, err := a.compileGlob(excludeGlob)
+		if err != nil {
+			return nil, fmt.Errorf("compile exclude glob: %s", err)
```

Comment:
it would be good to add the actual global that failed to the error message: `compile exclude glob %q: %s`